PluginCore/PluginCore/Resources/zh_CN.resx

```diff
@@ -6120,4 +6120,16 @@ AND - All conditions are met.</value>
     <value>The default column to group results by.</value>
 	<comment>Added after 5.2.1</comment>
   </data>
+  <data name="LintingHelper.Description.LintOnSave" xml:space="preserve">
+    <value>Runs the available linters when a file is saved.</value>
+	<comment>Added after 5.2.1</comment>
+  </data>
+  <data name="LintingHelper.Description.LintOnOpen" xml:space="preserve">
+    <value>Runs the available linters when a file is opened.</value>
+	<comment>Added after 5.2.1</comment>
+  </data>
+  <data name="LintingHelper.Label.RunLinters" xml:space="preserve">
+    <value>Run Linters</value>
+	<comment>Added after 5.2.1</comment>
```

Comment:
Yep, if you check there are already a lot of entries like that heh. It was planned to make 5.2.1 happen, but sadly due to lack of time it never happened.